Lib/turtledemo/snake.py

```diff
@@ -0,0 +1,188 @@
+# Simple Snake Game in Python 3 for Beginners
+# Coded by Ehsonjon Gadoev
+
+__author__ = "Ehsonjon Gadoev (github.com/icoder-new)"
+__credits__ = ["Guido van Rossum, for an excellent programming language!", "Ilyosiddin Kalandar, for helping to made this game!"]
+
+import turtle
+from time import sleep
+from random import randint, choice
+
+delay = 0.1
+
+# Score
+score = 0
+high_score = 0
+
+# Set up the screen
+wn = turtle.Screen()
+wn.title("Snake Game by Ehsonjon Gadoev")
+wn.bgcolor("green")
+wn.setup(width=610, height=610)
+wn.tracer(0) # Turns off the screen updates
+
+# Snake head
+head = turtle.Turtle()
+head.speed(0)
+head.shape("square")
+head.color("black")
+head.penup()
+head.goto(0,0)
+head.direction = "stop" 
+
+# Snake food
+food = turtle.Turtle()
+food.speed(0)
+food.shape("circle")
+food.color("red")
+food.penup()
+food.goto(0,100)
+
+segments = []
+
+# Pen
+pen = turtle.Turtle()
+pen.speed(0)
+pen.shape("square")
+pen.color("white")
+pen.penup()
+pen.hideturtle()
+pen.goto(0, 260)
+pen.write("Score: 0  High Score: 0", align="center", font=("Courier", 24, "normal"))
+
+# Functions
+def go_up():
+    if head.direction != "down":
+        head.direction = "up"
+
+def go_down():
+    if head.direction != "up":
+        head.direction = "down"
+
+def go_left():
+    if head.direction != "right":
+        head.direction = "left"
+
+def go_right():
+    if head.direction != "left":
+        head.direction = "right"
+
+def move():
```

Comment:
The 'if's should be 'if-elif....'.  I actually believe the move could be done with dict lookup, but I need to check the turtle doc to be sure.